src/ts/accordion.ts

```
@@ -0,0 +1,81 @@
+
+    const originalHeaderY =
+      window.scrollY + details.getBoundingClientRect().top;
+    if (originalHeaderY >= window.scrollY) return;
+
+    event.preventDefault();
+
+    const stickyTop = parseFloat(window.getComputedStyle(summary).top) || 0;
+    const targetScrollY = Math.max(0, Math.round(originalHeaderY - stickyTop));
+
+    let animationFrameId = 0;
+    let timeoutId: number | undefined;
```

ESLint

[error] 'timeoutId' is never reassigned. Use 'const' instead.

(prefer-const)